rpcmetrics/observer_test.go

```diff
@@ -142,13 +141,13 @@ func TestTags(t *testing.T) {
 
 	testCases := []tagTestCase{
 		{key: "something", value: 42, metrics: []u.ExpectedMetric{
-			{Name: "requests", Value: 1, Tags: tags("error", "false")},
+			{Name: "operation", Value: 1, Tags: tags("result", "ok")},
 		}},
 		{key: "error", value: true, metrics: []u.ExpectedMetric{
-			{Name: "requests", Value: 1, Tags: tags("error", "true")},
+			{Name: "operation", Value: 1, Tags: tags("result", "err")},
 		}},
 		{key: "error", value: "true", metrics: []u.ExpectedMetric{
-			{Name: "requests", Value: 1, Tags: tags("error", "true")},
+			{Name: "operation", Value: 1, Tags: tags("result", "err")},
 		}},
 	}
 
```

Comment:
nit: using "i" as the name for the first number in a status code makes this test hard to read.  Personally I would prefer. "http_status_range"